docs/content/main/configuration.md

```diff
@@ -220,6 +220,13 @@ The `<windows>` config should look something like this:
 The window block contains multiple elements to configure the window.
 - `<geometry>` is used to specify the position and size of the window.
 - `<widget>` will contain the widget that is shown in the window.
+- `<struts>` Tells the window manager to reserve space at the edge of the screen (for a status bar or panel sort of widget).
+  The `left`, `right`, `top` and `bottom` attributes tell the WM how far to set struts away from that edge of the screen
+  The `left_start_y`, `left_end_y`, `right_start_y`, `right_end_y`, `top_start_x`, `top_end_x`, `bottom_start_x` and `bottom_end_x`
```

Comment:
It would be good to mention that it is not required to set all of these, and to explain how only setting for example top and bottom would behave, etc

Reply:
also, a link to the X documentation describing how the _NET_WM_STRUT(_PARTIAL) works would probably be a good idea